ONSdigital/dp-frontend-renderer
# Feature/json ld implementation

### What

Created partials that build `JSONLD` tags that are used on certain pages. In the case of the Renderer, these tags are present on dataset pages (legacy and filterable), as well as the dataset timeseries page.

### How to review

- Ensure that `dp-frontend-dataset-controller` is running on branch `feature/json-ld-implementation`
- Ensure that `dp-frontend-models` has been pulled and both renderer and the dataset controller are using `v1.5.0` of the models
- run the renderer with `make debug ENABLE_JSONLD_CONTROL=true`
- run `dp-frontend-router` with `make debug DATASET_ROUTES_ENABLED=true`

- Once all running, visit the relevant pages - legacy dataset landing page, filterable dataset landing page and dataset timeseries page. On each page you should see within the `<head>` a script with `type="application/ld+json"` - review the contents of that script
- Ensure that the json is valid
- Ensure that the fields broadly align with that which are in GTM

- Check that code changes make sense

### Who can review

Anyone but me
